extension-style/src/main/AndroidManifest.xml

```diff
@@ -1 +1 @@
-<manifest package="com.mapbox.maps.plugin.style" />
+<manifest package="com.mapbox.maps.extension.style" />
```

Comment:
Fixed the package name for the style extension, this shouldn't introduce breaking change, as described in  https://developer.android.com/guide/topics/manifest/manifest-element#package

> While building your app into the an application package (APK), the build system uses the package attribute for two things:
> 
> * It applies this name as the namespace for your app's generated R.java class (used to access your [app resources](https://developer.android.com/guide/topics/resources/overview)).
> For example, if package is set to "com.example.myapp", the R class is created at com.example.myapp.R.
> 
> * It uses this name to resolve any relative class names that are declared in the manifest file.
> For example, if package is set to "com.example.myapp", an activity declared as <activity android:name=".MainActivity"> is resolved to be com.example.myapp.MainActivity.

And there's no resources defined for the style extension or relative classes declared in the manifest file.